src/App.js

```diff
@@ -1,24 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import BooksList from './containers/BooksList';
+import BookForm from './containers/BooksForm';
 
 function App() {
```

Comment:
Let move `App.js` inside component folder because it is one of the presentational components right?

![image](https://user-images.githubusercontent.com/31897434/74705649-eb604580-5214-11ea-8b58-b7c9ea068ab3.png)